kuma/core/helpers.py

```diff
@@ -150,8 +150,8 @@ def entity_decode(str):
 
 
 @register.function
-def inlinei18n(locale):
-    return StaticI18nJob().get(locale)
+def statici18n(locale):
+    return statici18n_orig(locale)
```

Comment:
You can just do `register.function(statici18n)` here and import the original without a separate name